pkg/git/common/common.go

```diff
@@ -0,0 +1,49 @@
+/*
+Copyright 2020 The Flux CD contributors.
```

Comment:
```suggestion
Copyright 2020 The Flux authors
```

(Change applies to all files)